Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,327 @@
+<?php
+
+/**
+ * File containing the main context class for PlatformUI.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context;
+
+use EzSystems\BehatBundle\Context\Browser\Context;
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit_Framework_Assert as Assertion;
+
+class PlatformUI extends Context
+{
+    use SubContext\CommonActions;
+
+    /**
+     * PlatformUI relative URL path
+     *
+     * @var string
+     */
+    private $platformUiUri;
+    protected $shouldBeLoggedIn;
+    protected $driver;
+
+    /**
+     * @Given I go to homepage
+     */
+    public function goToPlatformUi()
+    {
+        $this->visit( $this->platformUiUri );
+    }
+
+    /**
+     * @Given I go to PlatformUI app with username :user and password :password
+     */
+    public function goToPlatformUiAndLogIn( $username, $password )
+    {
+        // Given I go to PlatformUI app
+        $this->goToPlatformUi();
+        //wait fos JS
+        $this->waitForJs();
+        // And I fill in "Username" with "admin"
+        $this->fillFieldWithValue( 'Username', $username );
+        //And I fill in "Password" with "publish"
+        $this->fillFieldWithValue( 'Password', $password );
+        //And I click on the "Login" button
+        $this->iClickAtButton( 'Login' );
```

Comment:
The iClickAtButton funtion belong to BehatBundle browser context, but I can change it.